javascript/bower.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "ola",
+  "version": "0.0.1",
```

Comment:
Hmm, in which case I'd be tempted to make it match the OLA version anyway, even for the future.